lib/metasploit/framework/login_scanner/kerberos.rb

```diff
@@ -23,12 +23,22 @@ def attempt_login(credential)
           }
 
           begin
-            res = send_request_tgt(
-              server_name: server_name,
-              client_name: credential.public,
-              password: credential.private,
-              realm: credential.realm
-            )
+            begin
+              res = send_request_tgt(
+                server_name: server_name,
+                client_name: credential.public,
+                password: credential.private,
+                realm: credential.realm,
+                offered_etypes: [Rex::Proto::Kerberos::Crypto::Encryption::RC4_HMAC]
```

Comment:
Was wondering if we would request normal encryption types by default, and maybe only in the ASREP roasting scenario we try to downgrade to RC4_HMAC? 🤔 

That way we're not making double requests against modern boxes that reject RC4_HMAC etype

Reply:
I've implemented that approach with the new commit.